locale/pl/LC_MESSAGES/client.po

```diff
@@ -448,53 +448,59 @@ msgstr ""
 
 #: app/scripts/lib/strings.js:29
 msgid "Confirm new sign-in to Firefox"
-msgstr ""
+msgstr "Potwierdź nowe logowanie do Firefoksa"
 
 #: app/scripts/lib/strings.js:30
 msgid "You signed in to Firefox from %(device)s, %(location)s."
-msgstr ""
+msgstr "Zalogowano do Firefoksa z urządzenia %(device)s, %(location)s."
 
 #: app/scripts/lib/strings.js:31
 msgid "You signed in to Firefox from %(device)s."
-msgstr ""
+msgstr "Zalogowano do Firefoksa z urządzenia %(device)s."
 
 #: app/scripts/lib/strings.js:32
 msgid ""
 "For added security, please confirm this sign-in to begin syncing with "
 "%(device)."
 msgstr ""
+"Z powodów bezpieczeństwa prosimy potwierdzić to logowanie, aby zacząć "
+"synchronizację z urządzeniem %(device)."
 
 #: app/scripts/lib/strings.js:33
 msgid "Alternatively:"
-msgstr ""
+msgstr "Lub można:"
```

Comment:
I copied the translation from server.po.

I think "Można również" suggests AND/OR, while I'm guessing it's just OR.